RELEASE.md

```diff
@@ -1,5 +1,5 @@
 # Release 0.3.0
-- Added a method `Model.remove_fixable_singularities` to remove fixable singularities in the model's equations.
```

Comment:
Did you mean to remove the old release info?!

Reply:
eeuh uups no!